tests/models/bert_japanese/test_tokenization_bert_japanese.py

```diff
@@ -425,25 +424,3 @@ def test_tokenizer_bert_japanese(self):
         EXAMPLE_BERT_JAPANESE_ID = "cl-tohoku/bert-base-japanese"
         tokenizer = AutoTokenizer.from_pretrained(EXAMPLE_BERT_JAPANESE_ID)
         self.assertIsInstance(tokenizer, BertJapaneseTokenizer)
-
-
-class BertTokenizerMismatchTest(unittest.TestCase):
-    def test_tokenizer_mismatch_warning(self):
-        EXAMPLE_BERT_JAPANESE_ID = "cl-tohoku/bert-base-japanese"
-        with self.assertLogs("transformers", level="WARNING") as cm:
-            BertTokenizer.from_pretrained(EXAMPLE_BERT_JAPANESE_ID)
-            self.assertTrue(
-                cm.records[0].message.startswith(
-                    "The tokenizer class you load from this checkpoint is not the same type as the class this function"
-                    " is called from."
-                )
-            )
-        EXAMPLE_BERT_ID = "bert-base-cased"
-        with self.assertLogs("transformers", level="WARNING") as cm:
-            BertJapaneseTokenizer.from_pretrained(EXAMPLE_BERT_ID)
-            self.assertTrue(
-                cm.records[0].message.startswith(
-                    "The tokenizer class you load from this checkpoint is not the same type as the class this function"
-                    " is called from."
-                )
-            )
```

Comment:
This test should not be deleted. It is still passing.

Reply:
Does it? I dropped it because it looked like type checking that would fail now that the japanese one doesn't subclass, I'll see if it actually runs if I change the import. The import that I removed in this same file has to come from the original file because it's removed from bert japanese due to the nature of this PR